v4/feature/profile/src/main/java/exchange/dydx/trading/feature/profile/DydxProfileView.kt

```diff
@@ -81,6 +84,11 @@ object DydxProfileView : DydxComponent {
                 item(key = "buttons") {
                     DydxProfileButtonsView.Content(Modifier.padding(horizontal = ThemeShapes.HorizontalPadding))
                 }
+                if (state?.hasAlerts == true) {
+                    item(key = "vault") {
```

Comment:
i think `key` should be `alerts` ?

Reply:
Oh yeah.. that's right.. I will update in the next PR